### PR TITLE
PM-5263  - TokenService needs to actually use secure storage

### PR DESCRIPTION
**Pull request by JaredSnider-Bitwarden**
## Type of change

```
- [X] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Fix `TokenService` not using `SecureStorageService` injection token so data wasn't truly being saved in secure storage. 

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/desktop/src/main.ts [hunk 1]
```diff
@@ -147,11 +148,14 @@ export class Main {
 
     this.environmentService = new EnvironmentService(stateProvider, accountService);
 
+    // Note: secure storage service is not available and should be called in the main background process.
```
Comment (justindbaur) [inline apps/desktop/src/main.ts:151]:
```suggestion
    // Note: secure storage service is not available and should not be called in the main background process.
```

**Review by justindbaur — CHANGES_REQUESTED** [review]

**Review by justindbaur — APPROVED** [review]

**Comment by BetoMees** [comment]
this was broke the login on desktop (windows)

**Comment by JaredSnider-Bitwarden** [comment]
> This has broken the login functionality on desktop (Windows).

@BetoMees , what error are you seeing on login so I can confirm the behavior I'm seeing is what you saw?

**Comment by BetoMees** [comment]
I'm not receiving any error, the token response is working just fine but doesn't login in.
Usually the application initialize without the need to enter the password but now is requesting.
I'm looking ass well and any news I'll comment here

**Comment by BetoMees** [comment]
Here is the fix #8364

**Pull request merged**